.github/workflows/pr-ci.yml

```diff
@@ -66,6 +70,7 @@ jobs:
         run: |
           git fetch origin ${{ matrix.commit }} ${{ github.event.pull_request.merge_commit_sha }}
           git merge --no-ff --no-edit ${{ matrix.commit }}
+          git submodule update --init --recursive
```

Comment:
what does this fix exactly? i don't get the commit title

Reply:
I improved the wording, but also removed an unnecessary pulling of the submodules in that commit. PTAL